### ISSUE TITLE
Docker stats collection doesn't work with kubernetes

### ISSUE THREAD

**Issue by tomwilkie** [weaveworks/scope#1291]
I think its because kubernetes runs a pause container (which has stats) and then other containers which share various namespaces:

![screen shot 2016-04-13 at 12 47 53](https://cloud.githubusercontent.com/assets/444037/14492244/f9fb5f74-0175-11e6-8262-75bd6673ce93.png)

**Comment by tomwilkie** [comment]
Yeah seems to be the case:

```
Toms-MacBook-Pro:docker twilkie$ docker ps | grep scope 
f0c6ddc262cd        weaveworks/scope                                     "/home/weave/entrypoi"   28 minutes ago      Up 28 minutes                                k8s_weave-scope-probe.a8eb0e36_weave-scope-probe-o7hbj_default_6351451c-016a-11e6-a58b-aac114fc48cf_cf9eb212
9b27b53a09dd        beta.gcr.io/google_containers/pause:2.0              "/pause"                 28 minutes ago      Up 28 minutes                                k8s_POD.b67ce16c_weave-scope-probe-o7hbj_default_6351451c-016a-11e6-a58b-aac114fc48cf_8cc78d96
aadda282399c        sha256:95e0d                                         "/home/weave/entrypoi"   About an hour ago   Up About an hour                             k8s_weave-scope-app.7747597c_weave-scope-app-88si2_default_ee71d9f7-0164-11e6-a58b-aac114fc48cf_e8f6d2b5
87c998b9eb5f        beta.gcr.io/google_containers/pause:2.0              "/pause"                 About an hour ago   Up About an hour    0.0.0.0:4040->4040/tcp   k8s_POD.3704032f_weave-scope-app-88si2_default_ee71d9f7-0164-11e6-a58b-aac114fc48cf_b2e908fd

# docker stats on the actual probe container
Toms-MacBook-Pro:docker twilkie$ docker stats f0c6ddc262cd
CONTAINER           CPU %               MEM USAGE / LIMIT   MEM %               NET I/O             BLOCK I/O
f0c6ddc262cd        0.00%               0 B / 0 B           0.00%               0 B / 0 B           0 B / 0 

# docker stats on the pause container
Toms-MacBook-Pro:docker twilkie$ docker stats 9b27b53a09dd
CONTAINER           CPU %               MEM USAGE / LIMIT     MEM %               NET I/O             BLOCK I/O
9b27b53a09dd        0.00%               1.528 MB / 4.143 GB   0.04%               0 B / 0 B           0 B / 0 B
```

**Comment by tomwilkie** [comment]
Docker issue: https://github.com/docker/docker/issues/21848

Fix is in Docker 1.11.
